tracing-subscriber/src/fmt/format/pretty.rs

```diff
@@ -247,21 +257,34 @@ impl Default for PrettyFields {
 impl PrettyFields {
     /// Returns a new default [`PrettyFields`] implementation.
     pub fn new() -> Self {
-        Self { ansi: true }
+        // By default, don't override the `Writer`'s ANSI colors
+        // configuration. We'll only do this if the user calls the
+        // deprecated `PrettyFields::with_ansi` method.
+        Self { ansi: None }
     }
 
+    #[deprecated(
+        since = "0.3.3",
+        note = "Use `fmt::Subscriber::with_ansi` or `fmt::Collector::with_ansi` instead."
+    )]
     /// Enable ANSI encoding for formatted fields.
```

Comment:
_very_ silly minor nit: normally, we put the attributes after the doc comment:
```suggestion
    /// Enable ANSI encoding for formatted fields.
    #[deprecated(
        since = "0.3.3",
        note = "Use `fmt::Subscriber::with_ansi` or `fmt::Collector::with_ansi` instead."
    )]
```